src/java.desktop/windows/native/libawt/windows/ThemeReader.cpp

```diff
@@ -90,7 +90,7 @@ typedef HRESULT (__stdcall *PFNGETTHEMETRANSITIONDURATION)
                 (HTHEME hTheme, int iPartId, int iStateIdFrom, int iStateIdTo,
                  int iPropId, DWORD *pdwDuration);
 
-static PFNOPENTHEMEDATA OpenThemeDataFunc = NULL;
+static PFNOPENTHEMEDATAFORDPI OpenThemeDataFuncForDpi = NULL;
```

Comment:
All other function have the `-Func` postfix, we should follow the naming convention.